resources/data/tagging-preset.xsd

```diff
@@ -121,7 +121,7 @@
     <complexType name="item">
         <annotation>
             <documentation>
-                Every item is one annotation set to select from. name is required, type and preset_name_label are recommended, icon and name_template are optional attributes.
+                Every item is one annotation set to select from. name is required, type and preset_name_label are recommended and icon, name_template, regions and exclude_regions are optional attributes.
```

Comment:
Can you reread this sentence. I don't think it flows well. Maybe replace the first `and` with `. `.